src/containers/pages/InterventionPlan/NewPlan/General/tests/index.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { mount, shallow } from 'enzyme';
```

Comment:
This replaces `NewPlan/General/test/index.test.tsx ` which was removed.  (just the test directory was renamed to tests)